tasks-by-users/AbbasShaikh0304/07-Responsive Css Grid/style.css

```diff
@@ -0,0 +1,112 @@
+body {
+  font-family: "Poppins", sans-serif;
+}
+:root {
+  --main-radius: 5px;
+
+  --main-padding: 5px;
+}
+
+.container {
+  display: grid;
+
+  height: 100vh;
+
+  grid-template: 0.2fr 1.5fr 1.2fr 0.8fr / 1fr 1fr 1fr 1fr;
```

Comment:
```suggestion
  grid-template: 0.2fr 1.5fr 1.2fr 0.8fr / repeat(4, 1fr);
```

Reply:
Thankyou for you suggestions. Commited the Changes @AlishaPrasad